test/app.spec.js

```diff
@@ -1,18 +1,22 @@
 const assert = require('assert');
+const Cacheman = require('cacheman');
 const chai = require('chai');
 const chaiHttp = require('chai-http');
-const config = require('config');
 const reqPromise = require('request-promise');
 const sinon = require('sinon');
 
-const app = require('../src/app');
+const { Messenger } = require('../src/app');
 
 chai.use(chaiHttp);
 
 describe('app', () => {
-  const messenger = new app.Messenger();
+  let messenger;
   let session;
 
+  before(() => {
```

Comment:
this is a refactor following the best practices laid out in https://github.com/CondeNast-Copilot/autopilot/wiki/Tests,-Scope,-and-State-Bleed